File: src/Hoops.Core/Models/Color.cs
using System;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using Hoops.Core.Interface;

namespace Hoops.Core.Models
{
    public partial class Color : IAuditable
    {
        [Key]
        [DatabaseGenerated(DatabaseGeneratedOption.None)]
        [Column("ID")]
        public int ColorId { get; set; }
        [Column("CompanyID")]
        public int? CompanyId { get; set; }
        public string ColorName { get; set; }
        public bool? Discontinued { get; set; }
        public DateTime? CreatedDate { get; set; }
        public int? CreatedUser { get; set; }
        public DateTime? ModifiedDate { get; set; }
        public int? ModifiedUser { get; set; }
    }
}

File: hoops.ui/src/domain/color.ts
export class Color {
  public colorId!: number;
  public companyId: number = 1;
  public colorName!: string;
  public discontinued: boolean | null = null;
  public createdDate: Date | string | null = null;
  public createdUser: number | null = null;
  public modifiedDate: Date | string | null = null;
  public modifiedUser: number | null = null;
}
